File: modules/react-native-sample/ios/ViewManagers/PriceBoardListView/PriceBoardListViewManager.m
#import <Foundation/Foundation.h>

#import <React/RCTUtils.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_REMAP_MODULE(PriceBoardListView, PriceBoardListViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(onPressHandle, RCTDirectEventBlock);

@end

File: modules/react-native-sample/ios/ViewManagers/PriceBoardListView/PriceBoardListViewManager.swift
import Foundation

@objc(PriceBoardListViewManager)
class PriceBoardListViewManager: RCTViewManager {
    
    override var methodQueue: dispatch_queue_t! {
        return .main
    }
    
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
    
    override func view() -> (PriceBoardListView) {
        return PriceBoardListView()
    }
}
